### DISCUSSION TITLE
有没有指定一个网段，让这个网段的设备都能免密登录的功能啊

### DISCUSSION THREAD

**Discussion by m0eak** [hslr-s/sun-panel#258] [General]
类似qb一样的功能，指定192.168.xx.0/24，192.168.xx.x/24的设备都能免密登录，部署在局域网每次输入密码有点麻烦，

**Comment by Chenghao999** [comment]
赞同我认为可以增加 白名单不需要验证码登陆，以及可以设置登出时间
